examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    region_name = "us-east-1"
</pre><p>Is it temporary? If not, why is the region hard coded? Can users just use default region?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I was previously experiencing some frictions where my notebook is in east-1 but the jobs are sent to west-1. That's why I think it might be a good idea to specify the region. I will add a line "Adjust the region as needed"